Update Manual JWT verification page (#925)
clerk/clerk-docs: docs/backend-requests/handling/manual-jwt.mdx

For every request, you must validate its token to make sure it hasn't expired and it is authentic (i.e. no malicious user tried to tamper with it). If these validations pass, then it means that the user is authenticated to your application and you should consider them signed in.

> The `authenticateRequest()` method from Clerk's JavaScript Backend SDK does all of this for you. It accepts the request object and authenticates the session token in it. See the [reference page](/docs/references/backend/sessions/authenticate-request) for example usage and more information.

Reviewer: I have some notes that should apply before merging this PR.
- I believe that `authenticate-request` should not be part of the `/docs/references/backend/sessions` as it's not exactly related to  the `sessions` and it's definitely not related to our backend endpoints grouped in that url path.
- code example in [authenticate request page](https://clerk.com/docs/references/backend/sessions/authenticate-request) should be updated to use `@clerk/backend` package instead of nextjs
- I think we should keep a code example of how `authenticateRequest()` can be used in this page, otherwise does this page make much sense? It's just an intro and a ref to another page.

cc: @alexisintech , @octoper

Reviewer 2: yeah I agree - this page will definitely get reworked in the upcoming IA. for now, I've made the changes you requested :)

> 1. Using the Backend API in JSON Web Key Set (JWKS) format at the following endpoint [https://api.clerk.com/v1/jwks](https://clerk.com/docs/reference/backend-api/tag/JWKS#operation/GetJWKS).

Reviewer 2: ```suggestion
1. Use the Backend API in JSON Web Key Set (JWKS) format at the following endpoint [https://api.clerk.com/v1/jwks](https://clerk.com/docs/reference/backend-api/tag/JWKS#operation/GetJWKS).
```

2. Using the Frontend API in JSON Web Key Set (JWKS) format at the following endpoint `https://<YOUR_FRONTEND_API>/.well-known/jwks.json`. This can be obtained from the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page. At the left sidebar click on **Show JWT public key** and in the model that will show up you can find and copy the **JWKS URL**.

> 3. Using the PEM public key provided in the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page. At the left sidebar click on **Show JWT public key** and in the model that will show up you can find and copy the **PEM Public Key**.. This option should only be used as a fallback for when the first two options are not available.

Reviewer 3: ```suggestion
3. Using the PEM public key provided in the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page. In the left sidebar, select **Show JWT public key**. In the modal that appears copy the **PEM Public Key**. This option should only be used as a fallback for when the first two options are not available.
```

## Example usage

> In the example below, the [@clerk/backend](https://www.npmjs.com/package/@clerk/backend) is used to verify the token signature. This example uses Next.js route handlers just to showcase how this works, you can use `verifyToken` with any JS framework or no framework at all.

Reviewer 3: ```suggestion
The following example demonstrates how to use [@clerk/backend](https://www.npmjs.com/package/@clerk/backend) to verify the token signature. Although this example uses Next.js route handlers, you can use `verifyToken` with any JS framework or no framework at all.
```
